pkg/java/pom/parse.go

```diff
@@ -156,7 +156,9 @@ func (p *parser) parseRoot(root artifact) ([]types.Library, []types.Dependency,
 		for _, relativePath := range result.modules {
 			moduleArtifact, err := p.parseModule(result.filePath, relativePath)
 			if err != nil {
-				return nil, nil, xerrors.Errorf("module error (%s): %w", relativePath, err)
+				log.Logger.Error(xerrors.Errorf("module error (%s): %w - Some modules may be missing. "+
+					"Skipping to the next module", relativePath, err))
```

Comment:
We usually use `Debug` for wrong part of dependency. What do you think about this:
```suggestion
				log.Logger.Debugf("Unable to parse %q module: %s", result.filePath, err)
```